UI/views/schedule.html

```diff
@@ -27,9 +27,9 @@ <h3 class="colHeader"><span class="sr-only">Schedule</span>
          </div>
       </div>
       <div class="alert alert-info" ng-show="!filteredItems.length  && !loadingSchedule">
-      <p ng-if="scheduleStatus.ctools.status !==200 && scheduleStatus.canvas.status !==200">CTools and Canvas unreachable.</p>
-      <span ng-show="(scheduleStatus.ctools.status ==200 || scheduleStatus.canvas.status ==200) && schedule.length!==0">{{showWhen | noItemsFoundForTime}}</span>
-      <span ng-show="schedule.length===0 && (scheduleStatus.ctools.status ===200 || scheduleStatus.canvas.status ===200)">No items due in CTools or Canvas.</span></div>
+        <span ng-show="schedule.length !==0">{{showWhen | noItemsFoundForTime}}</span>
+        <span ng-show="schedule.length ===0">No items due in CTools or Canvas.</span>
```

Comment:
Is this right if we just didn't get information for one of CTools / Canvas?  What if one them didn't return information?